lib/nadia.ex

```diff
@@ -39,7 +39,7 @@ defmodule Nadia do
   force a reply from the user - `Nadia.Model.ReplyKeyboardMarkup` or
   `Nadia.Model.ReplyKeyboardHide` or `Nadia.Model.ForceReply`
   """
-  @spec send_message(integer, binary, [{atom, any}]) :: {:ok, Message.t()} | {:error, Error.t()}
+  @spec send_message(integer | binary, binary, [{atom, any}]) :: {:ok, Message.t()} | {:error, Error.t()}
```

Comment:
hey @dog-64 , just curious, this is kind of a significant change, what stays behind it could you please give more details? thanks a lot!